graphqlws/internal/connection/connection.go

```diff
@@ -165,7 +166,7 @@ func (conn *connection) readLoop(ctx context.Context, send sendFunc) {
 				continue
 			}
 			send("", typeConnectionAck, nil)
-                       
+			header =msg.Payload
```

Comment:
Run `go fmt` on this. It is not formatted properly.

Reply:
formatted properly.